cmd/auth/main.go

```diff
@@ -324,6 +325,7 @@ func main() {
 	loginidhandler.AttachAddLoginIDHandler(apiRouter, authDependency)
 	loginidhandler.AttachRemoveLoginIDHandler(apiRouter, authDependency)
 	loginidhandler.AttachUpdateLoginIDHandler(apiRouter, authDependency)
+	challengehandler.AttachCreateHandler(apiRouter, authDependency)
```

Comment:
Checking for disabled Auth API is performed in `AuthAPIXXX` authz policies, so it would not be affected. Do you mean using a path of `/challenge` instead of `/_auth/challenge`? It would be strange since it would seems like a user-accessible endpoint rather than API endpoint.

Reply:
The prefix `/_auth/` is well understood as the legacy Auth API. I tend not to add new things to it.

The token endpoint does not have `/_auth/` as well but it is really non-user-facing endpoint.

Reply:
Token endpoint is in `/oauth2/` which can be understood as an endpoint specified by OAuth2.

How about `/api/v1/challenge`?

Reply:
The challenge is part of the protocol of the new grant type though it is not specified by OAuth2. Maybe `/oauth2/x/challenge`?